spec/models/user_spec.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id         :integer          not null, primary key
+#  first_name :string
+#  last_name  :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it 'has a valid factory' do
```

Comment:
I am not a fan of this type of tests. There are tools that check all factories are valid, but I don't know if we want it. 
The model will have tests that will use the factory themselves

Reply:
mmm, I'm also not a fan but I wanted to have at least one test and this one looked like a good option we can remove it after installing devise that for sure it's installation will trigger some tests. WDYT ?

Reply:
sure